puppet/modules/kubernetes/templates/kubelet-config.yaml.erb

```diff
@@ -0,0 +1,94 @@
+kind: KubeletConfiguration
+apiVersion: kubelet.config.k8s.io/v1beta1
+clusterDNS:
+    - <%= @cluster_dns %>
+clusterDomain: <%= @cluster_domain %>
+<% if @pod_cidr -%>
+podCIDR: <%= @pod_cidr %>
+<% end -%>
+<% if @client_ca_file  -%>
+authentication:
+    x509:
+        clientCAFile: <%= @client_ca_file %>
+    anonymous:
+        enabled: false
+    webook:
+        enabled: true
+authorization:
+    mode: Webhook
+<% end -%>
+cgroupDriver: <%= @cgroup_driver %>
+cgroupRoot: <%= @cgroup_root %>
+<% if @cgroup_kube_name -%>
+kubeReservedCgroup: <%= @cgroup_kube_name %>
+kubeletCgroups: <%= @cgroup_kube_name %>
+kubeReserved:
+<% if !@cgroup_kube_reserved_cpu.nil? and @cgroup_kube_reserved_cpu != 'nil' -%>
+    cpu: <%= @cgroup_kube_reserved_cpu %>
+<% end -%>
+<% if !@cgroup_kube_reserved_memory.nil? and @cgroup_kube_reserved_memory != 'nil' -%>
```

Comment:
Needs to be `@_cgroup_kube_reserved_memory` and not `@cgroup_kube_reserved_memory`